build/toolchain/clang.gni

```diff
@@ -19,6 +19,9 @@ declare_args() {
 
   # Set this flag to cause bitcode to be marker only.
   bitcode_marker = false
+
+  # Set this flag to strip .so files.
+  stripped_symbols = true
```

Comment:
Can this be gated on `is_debug` please? These builds are not optimized and suitable for debugging. Stripping symbols will get in the way of this.

Reply:
Thanks for the review. I fixed it.